libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -910,7 +910,9 @@ eth::AssemblyPointer ContractCompiler::cloneRuntime()
 	a << Instruction::DELEGATECALL;
 	//Propagate error condition (if DELEGATECALL pushes 0 on stack).
 	a << Instruction::ISZERO;
-	a.appendJumpI(a.errorTag());
+	eth::AssemblyItem falseTag = a.appendJumpI();
```

Comment:
It's not available for class `Assembly` and I didn't want to add it since it is only used here. However, I'd be happy to add it if you want.